app/src/main/res/values/strings.xml

```diff
@@ -12,6 +12,20 @@
     <string name="dialog_action_cancel">Abbrechen</string>
     <string name="dialog_error_title">Sorry</string>
 
+    <string name="homework_name">Titel</string>
+    <string name="homework_course">Kurs:</string>
+    <string name="homework_private">Private Aufgabe (nur für mich sichtbar)</string>
+    <string name="homework_description">Aufgabenstellung</string>
+    <string name="homework_processingTime">Bearbeitungszeit:</string>
+    <string name="homework_dateDash">&#8211;</string>
+    <string name="homework_publicSubmissions">Schülerabgaben untereinander sichtbar</string>
```

Comment:
![image](https://user-images.githubusercontent.com/9141483/29176449-32e35ce4-7dec-11e7-9a66-b2daa373076c.png)
![image](https://user-images.githubusercontent.com/9141483/29176489-4809cc52-7dec-11e7-98f5-0b2ab401d6bb.png)

don't see the problem in using the plural